test/backend_test.in.cpp

```diff
@@ -4462,6 +4463,39 @@ NGRAPH_TEST(${BACKEND_NAME}, max_pool_2d_1channel_1image_strided)
               read_vector<float>(result));
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, max_pool_3d)
+{
+    Shape shape_a{64, 3, 7, 8, 10};
+    Shape window_shape{2, 3, 2};
+    auto move_strides = Strides{2, 3, 4};
+    Shape padding_below{5, 6, 4};
+    Shape padding_above{6, 4, 5};
+    auto A = make_shared<op::Parameter>(element::f32, shape_a);
+    auto B = make_shared<op::Parameter>(element::f32, shape_a);
+
+    auto cpu_f = make_shared<Function>(
+        make_shared<op::MaxPool>(A, window_shape, move_strides, padding_below, padding_above),
+        op::ParameterVector{A});
+    auto int_f = make_shared<Function>(
+        make_shared<op::MaxPool>(B, window_shape, move_strides, padding_below, padding_above),
+        op::ParameterVector{B});
+    test::Uniform<float> rng(0.0f, 1.0f);
+    vector<vector<float>> args;
+
+    for (shared_ptr<op::Parameter> param : int_f->get_parameters())
+    {
+        vector<float> tensor_val(shape_size(param->get_shape()));
+        rng.initialize(tensor_val);
+        args.push_back(tensor_val);
+    }
+    auto int_results = execute(int_f, args, "INTERPRETER");
+    auto cpu_results = execute(cpu_f, args, "CPU");
```

Comment:
Not sure but it looks like ${BACKEND_NAME} should be here:
`auto cpu_results = execute(cpu_f, args, "${BACKEND_NAME}");`